package.json

```diff
@@ -38,8 +30,10 @@
   "bugs": "https://github.com/npm/libnpmsearch/issues",
   "homepage": "https://npmjs.com/package/libnpmsearch",
   "dependencies": {
-    "figgy-pudding": "^3.5.1",
     "get-stream": "^4.0.0",
```

Comment:
Is this still used anywhere?  Should be replaced with a minipass thing instead, if so.